gateway/errors.go

```diff
@@ -119,16 +125,26 @@ func (h *ProtoErrorHandler) writeError(ctx context.Context, headerWritten bool,
 	if fields != nil {
 		restErr["fields"] = fields
 	}
+	if setStatusDetails {
+		restErr["code"] = statusCode
+		restErr["status"] = statusStr
+	}
 
-	errs, _ := errorsAndSuccessFromContext(ctx)
+	errs, _, overrideErr := errorsAndSuccessFromContext(ctx)
 	restResp := &RestErrs{
 		Error: errs,
 	}
-	restResp.Error = append(restResp.Error, restErr)
+	logrus.Infof("%v", overrideErr)
```

Comment:
I see you are adding a log here... 
I don't think we have added logs in the other parts of the tool. I think we should maybe add a little more info if we are going to include it. So that a developer can know where it is coming from.

Reply:
Whoops. I probably left that in while I was doing manual testing...